src/components/calcite-pagination/calcite-pagination.tsx

```diff
@@ -227,12 +226,12 @@ export class CalcitePagination {
           disabled={prevDisabled}
           onClick={this.previousClicked}
         >
-          <calcite-icon dir={dir} flipRtl icon="chevronLeft" scale={iconScale} />
+          <calcite-icon dir={dir} flipRtl icon="chevronLeft" scale="s" />
```

Comment:
Would this be considered a breaking change if we aren't honoring scale anymore?

Reply:
We are still honoring the scale for the overall size of the pagination component. This just has the size of the chevron icons at small for all sizes, but the text, padding, etc. are following the s, m, l scale.